x-pack/plugins/task_manager/server/task_store.test.ts

```diff
@@ -490,9 +491,13 @@ describe('TaskStore', () => {
           },
         ],
       } as estypes.MsearchResponse);
-      await expect(store.msearch([{}])).rejects.toThrowErrorMatchingInlineSnapshot(
-        `"Unexpected status code from taskStore::msearch: 429"`
-      );
+
+      try {
+        await store.msearch([{}]);
```

Comment:
**optional nit:** not sure if we need this to ensure the `catch` path goes through.

```suggestion
        await store.msearch([{}]);
        throw new Error('should have thrown');
```

Reply:
Updated in https://github.com/elastic/kibana/pull/197501/commits/241935d1b9c0e7dd17422d5eeafafd97b3cfb73e